client/state/action-types.js

```diff
@@ -317,6 +317,7 @@ export const LAYOUT_NEXT_FOCUS_SET = 'LAYOUT_NEXT_FOCUS_SET';
 export const LOGIN_REQUEST = 'LOGIN_REQUEST';
 export const LOGIN_REQUEST_FAILURE = 'LOGIN_REQUEST_FAILURE';
 export const LOGIN_REQUEST_SUCCESS = 'LOGIN_REQUEST_SUCCESS';
+export const LOAD_COMMENT = 'LOAD_COMMENT';
```

Comment:
our old paradigm was X_REQUEST, X_SUCCESS, X_ERROR. It made sense to have them that way since they were essentially logging things that already occured.  

In this case, i'm issuing a command to middleware to load something, so i think it makes sense to have the word LOAD come first.